python-lib/dku_neo4j/__init__.py

```diff
@@ -150,11 +157,11 @@ def load_relationships_from_csv(self, csv_file_path, columns_list, params):
 USING PERIODIC COMMIT
 LOAD CSV FROM 'file:///{csv_file_path}' AS line FIELDTERMINATOR '\t'
 WITH {definition}
-MERGE (src:`{params.source_node_label}` {source_node_primary_key_statement})
+MERGE (src:`{params.source_node_label}`{source_node_primary_key_statement})
```

Comment:
It would be clearer if the query templates could be moved to a separate file with all the constants.

Reply:
Good point, I will do that directly in the main branch (feature/13-new-python-driver) after I merge the other branches